lib/Button/CTALeafLike.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import ChevronNegativeIcon from 'lib/assets/icon-chevron-negative.svg';
+
+export const CTALeafLike = styled.a`
+  box-sizing: border-box;
+  width: 19.3125rem;
+  height: 6.25rem;
+  padding: 0 7rem 0 3.125rem;
+  border-top-right-radius: 6.25rem;
+  color: ${({ theme }) => theme.palette.text};
+  text-decoration: none;
+  font-size: 1.375rem;
+  text-align: left;
+  line-height: 1.65rem;
+  text-transform: uppercase;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: ${({ theme }) => theme.palette.secondary};
+  background-image: url(${ChevronNegativeIcon});
+  background-repeat: no-repeat;
+  background-position: right 3.75rem top 2.0625rem;
+  @media (max-width: 45rem) {
```

Comment:
Regarding the redefinition, it would be interesting to only define it on `lib` and reuse that same value on the landing itself, but since they use different styling stacks (:eyes:) I don't think that's possible.

Reply:
I'll check what we can do (later)